markdownlint.js

```diff
@@ -182,9 +212,11 @@ program
   .version(pkg.version)
   .description(pkg.description)
   .usage('[options] <files|directories|globs>')
+  .arguments('[options] <files|directories|globs>')
   .option('-f, --fix', 'fix basic errors (does not work with STDIN)')
   .option('-s, --stdin', 'read from STDIN (does not work with files)')
-  .option('-o, --output [outputFile]', 'write issues to file (no console)')
+  .option('-o, --output <outputFile>', 'write issues to file (no console)')
+  .option('-j, --junit [junitFile]', 'write issues to file in junit format as well as to the console')
```

Comment:
Is “junit” the preferred capitalization?

Reply:
looks like `JUnit` is the correct name, but I'm thinking it makes sense to leave the option name and file name in the current case, and just change the descriptive text?

Reply:
fixed

Reply:
Agreed!